ocs_ci/helpers/managed_services.py

```diff
@@ -768,10 +768,13 @@ def verify_faas_provider_storagecluster(sc_data):
         sc_data (dict): storagecluster data dictionary
 
     """
-    log.info(
-        f"allowRemoteStorageConsumers: {sc_data['spec']['allowRemoteStorageConsumers']}"
-    )
-    assert sc_data["spec"]["allowRemoteStorageConsumers"]
+    if version.get_semantic_ocs_version_from_config() > version.VERSION_4_18:
+        pass
+    else:
+        log.info(
+            f"allowRemoteStorageConsumers: {sc_data['spec']['allowRemoteStorageConsumers']}"
+        )
+        assert sc_data["spec"]["allowRemoteStorageConsumers"]
```

Comment:
You can avoid else block I think
```suggestion
    if version.get_semantic_ocs_version_from_config() <= version.VERSION_4_18:
        log.info(
            f"allowRemoteStorageConsumers: {sc_data['spec']['allowRemoteStorageConsumers']}"
        )
        assert sc_data["spec"]["allowRemoteStorageConsumers"]
```

Also, can you please explain to me that now you don't change value in SC like you've done before in:
ocs_ci/templates/ocs-deployment/provider-mode/ocs_storagecluster_updated.yaml

https://github.com/red-hat-storage/ocs-ci/compare/2958f000b6d92b05eb19ae70deeb2d80f01bb3f9..ecd1d70d35003a2b9dc58f9ae011c55495ee0096

Reply:
I will change it in another PR, this PR is on hold because more changes are required to StorageCluster cr and since I applied another StorageCluster here >> https://github.com/red-hat-storage/ocs-ci/pull/11620 I will add changes to that new file.
Please ignore this PR for a while

Reply:
Hi @petr-balogh, the rule is actually opposite of what you proposed. 
So now, when we have 4.19 version constant on master branch I can write it like 
```
if version.get_semantic_ocs_version_from_config() >= version.VERSION_4_19:
  do something
```